HSTracker/Logging/LogLine.swift

```diff
@@ -95,12 +95,6 @@ struct LogDate: Comparable, Equatable {
 		}
 	}
 	
-	static func == (lhs: LogDate, rhs: LogDate) -> Bool {
-		return
-			lhs.date == rhs.date &&
-				lhs.subseconds == rhs.subseconds
-	}
```

Comment:
Not sure I understand why you need to remove this.

Reply:
The general idea is that Swift 4.1 will automatically generates implementations for struct conforming Hashable & Equatable protocols.

ref: https://www.hackingwithswift.com/articles/50/whats-new-in-swift-4-1